nautobot/extras/jobs.py

```diff
@@ -175,7 +178,19 @@ def name(cls):
 
     @classproperty
     def description(cls):
-        return getattr(cls.Meta, "description", "")
+        return dedent(getattr(cls.Meta, "description", "")).strip()
+
+    @classproperty
+    def description_first_line(cls):
+        """
+        This returns a plain text string from the first line of a description
+        """
+        if cls.description:
+            Markdown.output_formats["plain"] = strip_markdown
+            md = Markdown(output_format="plain")
+            md.stripTopLevelTags = False
+            return md.convert(cls.description.splitlines()[0])
```

Comment:
I did this because the first line could be a heading in markdown and therefore make the job list table look a bit funky if that's the case and it's rendered as html. What do you think?

Reply:
description_first_line extracts and returns the plaintext from the markdown if it exists on the first line..